homeassistant/components/wemo.py

```diff
@@ -61,9 +61,8 @@ def stop_wemo(event):
 
     def discovery_dispatch(service, discovery_info):
         """Dispatcher for WeMo discovery events."""
-        # name, model, location, mac
-        model_name = discovery_info.get('model_name')
-        serial = discovery_info.get('serial')
+        model_name = discovery_info[1]
```

Comment:
Hmm - had to update to get it to run, must be picking up an incorrect library version for some reason. 

Will put back!